Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -2419,15 +2419,12 @@ private static void deleteMetadataDirectory(Process process) {
      * @param task Task for which symlinks are removed
      */
     public static void deleteSymlinksFromUserHomes(Task task) {
-        WebDav webDav = new WebDav();
-
-        for (Role role : task.getRoles()) {
-            for (User user : role.getUsers()) {
-                try {
-                    webDav.uploadFromHome(user, task.getProcess());
-                } catch (RuntimeException e) {
-                    Helper.setErrorMessage(e.getLocalizedMessage(), logger, e);
-                }
+        if (task.isTypeImagesWrite()) {
```

Comment:
The test must be extended to a check if `task.isTypeImagesRead()` as even there a symlink is created but only a read-only symlink.